Makefile

```diff
@@ -7,6 +7,7 @@ RELEASE_DIR := /tmp/release-$(RELEASE_TAG)
 PRE_RELEASE := "1"
 RELEASE_TYPE := $(shell if [ $(PRE_RELEASE) = "0" ] ; then echo release; else echo pre-release ; fi)
 GOLANGCI_BINARY=golangci-lint
+IPFS_GATEWAY_URL ?= https://ipfs.status.im/
```

Comment:
is it only for dev ? where will "infura" be set for prod?

Reply:
In the desktop/mobile repos I would assume.

Reply:
ah, right, i see https://github.com/status-im/status-desktop/pull/6965/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R191